new_core/screens/ingame.js

```diff
@@ -0,0 +1,41 @@
+import { input, state, map } from "../state.js"
+
+const TILE_SIDE_LENGTH = 16
+
+export function setup () {
+  map.width = 5
+  map.height = 5
+}
+
+export function update () {
+  if (input.CANCEL) {
+    state.next = "INITIAL"
```

Comment:
Reminder: We need to set this behind some kind of button to prevent one-click to quit the map. Fulfills the ticket, so no issues here.